executor/joiner.go

```diff
@@ -61,6 +61,11 @@ type joiner interface {
 	// matched with at lease one inner row.
 	tryToMatch(outer chunk.Row, inners chunk.Iterator, chk *chunk.Chunk) (matched bool, isNull bool, err error)
 
+	// tryToMatchOuters tries to join an outer row with a batch of inner rows.
+	// It's used when the join is an outer join and the hash table is built
```

Comment:
`an outer join and the hash table is built using the inner side.`
->
`an outer join and the hash table is built using the outer side.`